packages/slate/test/interfaces/Operation/inverse/moveNode/child-to-parent.js

```diff
@@ -0,0 +1,9 @@
+import { Operation } from 'slate'
+
+export const input = { type: 'move_node', path: [0, 2, 1], newPath: [0, 2] }
```

Comment:
this is "child to _replace_ parent"? 🤔

Reply:
Might be debating semantics in the english language, but it isn't replacing its parent, it is moving to the location fo where its parent currently is, which in turn shifts the location of its current parent before the move.

Reply:
Could we add this comment to the test cases? I know that's not a really pervasive convention at this time but I think it would help for posterity.

Reply:
What you're describing is what I meant by replacing, but you're correct that was the wrong way to put it. The comment you added does help.